README.md

```diff
@@ -898,14 +912,19 @@ are ready to actually deploy.
 
    invocation that it echoes.
 
-6. In the case that you need to reindex run the manual `reindex` job on the 
+6. In Zenhub, move all tickets from the pipeline representing the source
+   deployment of the promotion to the pipeline representing the target
+   deployment.
+
+7. In the case that you need to reindex run the manual `reindex` job on the 
    Gitlab pipeline representing the most recent build on the current branch.
 
-7. Once reindexing is complete announce this in the #data-wranging Slack
+8. Once reindexing is complete announce this in the #data-wranging Slack
```

Comment:
```suggestion
8. Once reindexing is complete announce this in the #data-wrangling Slack
```
or `#dcp-ops` if we no longer need to notify `#data-wrangling`

Reply:
Good catch but this section will soon be removed by @jessebrennan.

Reply:
Actually it had already been removed so my latest push does not have that change.